src/Core/src/Core/IBorderStroke.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Define how the Shape outline is painted on Layouts.
+	/// </summary>
+	public interface IBorderStroke : IStroke
```

Comment:
Should this maybe be `IStrokeShape : IShape, IStroke`?

Then ShapeDrawable can take `IShape` and then check for `shape is IStroke`? Then optionally add strokes.